### PR TITLE
Finishing unit testing

### PR DESCRIPTION
**Pull request by juanDuqueS**

### CODE REVIEW

File: back-end-yaz/src/server/controllers/Payment_itemsController.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Payment_itemsController {
 
     static async getAllPayment_items(req, res) {
         try {
-            const allPayment_items = await Payment_itemsService.getAllPayment_itemss();
+            const allPayment_items = await Payment_itemsService.getAllPayment_items();
```
Comment (alejandroVaron) [inline back-end-yaz/src/server/controllers/Payment_itemsController.js:10]: This fixs should not be done in another pr?

File: back-end-yaz/src/server/controllers/Payment_serviceController.js [hunk 2]
```diff
@@ -34,8 +34,9 @@ class Payment_serviceController {
 
     static async updatePayment_serviceById(req, res) {
         const { id } = req.params;
+        const alteredPayment_service = req.body;
         try {
-            const updatePayment_service = await Payment_serviceService.updatePayment_serviceById(id, alteredItem)
+            const updatePayment_service = await Payment_serviceService.updatePayment_serviceById(id, alteredPayment_service)
```
Comment (alejandroVaron) [inline back-end-yaz/src/server/controllers/Payment_serviceController.js:39]: This fix should not be done in another pr?

File: back-end-yaz/src/server/controllers/Payment_itemsController.js (hunk 1 above)
Comment (juanDuqueS) [reply back-end-yaz/src/server/controllers/Payment_itemsController.js:10]: No

File: back-end-yaz/src/server/controllers/Payment_serviceController.js (hunk 2 above)
Comment (juanDuqueS) [reply back-end-yaz/src/server/controllers/Payment_serviceController.js:39]: No

**Review by prometteo14 — APPROVED** [review]

**Review by alejandroVaron — APPROVED** [review]
